packages/microcosm/test/helpers.js

```diff
@@ -4,7 +4,7 @@ export function asTree(history) {
   function dig(focus) {
     let children = focus.children.map(dig)
 
-    return children.length ? [focus.tag, ...children] : focus.tag
+    return children.length ? [focus.key, ...children] : focus.key
   }
 
   return asciitree(dig(history.toJSON().tree))
```

Comment:
As a side note, this is a utility for serializing history into a tree visualization. I'm using this for snapshot testing the history structure:

```
    a
    |
    b
   / \
  c   d
```

Super useful!